### PR TITLE
added contributing docs

### PR DESCRIPTION
**Pull request by helios-ag**
Contributing docs

### CODE REVIEW

**Comment by makasim** [comment]
@javiereguiluz  Could you please review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Contribution Guidelines
+=======================
+
+First of all, each single contribution is appreciated, whether a typo fix,
+improved documentation, a fixed bug or a whole new feature.
+
+## Feature request
+
+If you think a feature is missing, please report it or implement it. If you report it, describe the more
```
Comment (javiereguiluz) [inline CONTRIBUTING.md]: `describe the more precisely` ->  `describe precisely`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+If you think you have detected a bug or a doc issue, please report it (open [issue][issue]) or even better fix it. If you report it,
+please be the more precise possible. Here a little list of required information:
+
+ * Symfony-standard fork which reproduces the bug.
+ * Precise description of the bug.
+ * Symfony version used.
+ * Bundle version used.
+
+ ## Bug fix
```
Comment (javiereguiluz) [inline CONTRIBUTING.md]: Please remove the white space at the beginning of this line.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+ Then, install the dependencies through [Composer][composer]
+
+ ``` bash
+ $ composer install
+ ```
+
+ When you're on the new branch with the dependencies, code as much as you want and when the fix is ready,
+ you will need to add tests and update the documentation. Everything is tested with
+ [PHPUnit][php-unit], documentation formatted with markdown under the `Resources/doc` directory.
```
Comment (javiereguiluz) [inline CONTRIBUTING.md]: `formatted with markdown` -> `formatted with reStructuredText`  (and maybe you can link to http://symfony.com/doc/current/contributing/documentation/format.html#restructuredtext)

**Comment by javiereguiluz** [comment]
Very nice! @helios-ag thanks for contributing this guide.

**Comment by helios-ag** [comment]
@javiereguiluz done

**Comment by javiereguiluz** [comment]
@helios-ag thank you! It looks nice :+1:

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+ ```
+
+ When you're on the new branch with the dependencies, code as much as you want and when the fix is ready,
+ you will need to add tests and update the documentation. Everything is tested with
+ [PHPUnit][php-unit], documentation formatted with [reStructuredText][rst] under the `Resources/doc` directory.
+
+ To run tests, use the following command:
+
+ ``` bash
+ $ phpunit
```
Comment (makasim) [inline CONTRIBUTING.md]: ./bin/phpunit

**Comment by helios-ag** [comment]
@makasim done

**Comment by makasim** [comment]
Thanks.

**Pull request merged**